manifests/profile/prometheus/exporter/node.pp

```diff
@@ -24,6 +24,7 @@
   Optional[String] $version = undef,
   Array $covered_datacenters = [],
   String $default_datacenter = 'default',
+  String $log_file = '/var/log/prometheus-node-exporter.log',
```

Comment:
Magic string appears twice; here and in prometheus_errors_total.rb. Is there any clean way to avoid that?

Reply:
Unfortunately, custom facts can't read hieradata, since the hieradata is generated by the puppetserver after the puppet agent runs facter. I did at least move this out of the parameter area, so it doesn't imply itself to be something that can or should be ever changed.

Reply:
Good to know. I like this change.